android/java/org/chromium/chrome/browser/app/BraveActivity.java

```diff
@@ -2702,4 +2724,30 @@ public void onSharedPreferenceChanged(
             BraveRelaunchUtils.askForRelaunch(currentActivity);
         }
     }
+
+    public void enterPipMode() {
```

Comment:
it doesn't seem right that we just keep dumping a bunch of random methods in BraveActivity, this can't be the right way to do things on android cc @SergeyZhukovsky

Reply:
Why it does not seem right? It's not a random method, it's a method in charge of entering picture-in-picture mode. Picture-in-picture mode affects the main screen, displaying the activity in a smaller window; so it's usually a good fit including it at activity level. 

From the official Android documentation: https://developer.android.com/design/ui/mobile/guides/home-screen/picture-in-picture
<img width="1178" alt="Picture-in-picture_ _ _Mobile_ _ _Android_Developers" src="https://github.com/user-attachments/assets/4d13b304-b171-47cb-bee9-2a6bdabee28b" />


May I ask you to elaborate @bridiver ?